woocommerce/woocommerce
# Add `@woocommerce/extend-cart-checkout-block` package

### All Submissions:






### Changes proposed in this Pull Request:

- Add the existing https://github.com/woocommerce/blocks-template to `packages/js` under `extend-cart-checkout-block`.
- Rename the package name in `package.json` to `@woocommerce/extend-cart-checkout-block`.
- Update the package's README to give more installation information.
- Add `project.json` file - Note that this package itself is not installable (well, it is but it won't do anything!) it only exists on npm so `@wordpress/create-block` can download the template files. That's why there are no executors described in this file.

We would like to include this package to give the WooCommerce Blocks boilerplate code a place to live rather than its own standalone repository.



### How to test the changes in this Pull Request:

I'm not really sure how we'd go about testing this, it doesn't add functionality to WooCommerce, and the package would need to be pushed to npm before we can verify it's still working as intended.

### Other information:




### Changelog entry

> Add `@woocommerce/extend-cart-checkout-block` to WooCommerce monorepo.

### FOR PR REVIEWER ONLY:
